DesignAutomationHandler2018/DesignAutomationHandler.cs

```diff
@@ -13,6 +16,36 @@ class DesignAutomationHandlerApp : IExternalCommand
         public Result Execute(ExternalCommandData commandData, ref string message, ElementSet elements)
         {
             var app = commandData.Application.Application;
+            string[] files = Directory.GetFiles(app.AllUsersAddinsLocation, "*.addin");
+            foreach (string file in files)
```

Comment:
I am not sue we will check all location where *.addin will be by `app.AllUsersAddinsLocation`.. Did you double check `readme`?

Reply:
I will double-check this information. However, the `README` says 
" `C:\ProgramData\Autodesk\Revit\Addins\XXXX\` " as the dir, which is this location.